### ISSUE TITLE
Adding Inverter BKW850-V2 but missing all datapoints

### ISSUE THREAD

**Issue by koogle1** [rospogrigio/localtuya#1857]
I am trying to connect my solar inverter BKW850-v2 in a Home Assistant setup.
The device is correctly detected (ID, Address; Secret) but I get the following error message when trying to add entities (here: 2 - total energy, 3 - actual power provided, 4 - actual temperature):
"_Connection to device succeeded but no datapoints found, please try again. Create a new issue and include debug logs if problem persists._"

Provide details about your environment.
--> see attached file config_entry-localtuya-01JDT7JX4S9VZ1AJ8VECY8VYGW.json
- [partially; power scale wrong; temperature missing] Does the device work using the Home Assistant Tuya Cloud component ?
- [not tested] Does the device work using the Tinytuya (https://github.com/jasonacox/tinytuya) command line tool ?
- [not sure; it ws working before but the version was not older than 5.2.1 and a downgrade to that version did not work either] Was the device working with earlier versions of localtuya ?
- [x - all working there] Are you using the Tuya/SmartLife App in parallel ?

## Steps to reproduce
Try to add new entities

## DP dump
see attachment

## Provide Home Assistant traceback/logs
see attachment
not sure where to implement those logger.set statements and what file to record; could not load configuration.yaml with statements inside :-(

## Additional information
Logging data obtained attached:
[config_entry-localtuya-01JDT7JX4S9VZ1AJ8VECY8VYGW.json](https://github.com/user-attachments/files/17963310/config_entry-localtuya-01JDT7JX4S9VZ1AJ8VECY8VYGW.json)

**Comment by TechGenius1234** [comment]
Just keep trying. i just kept subbmitting and it let thru
